UnitySDK/Assets/ML-Agents/Scripts/DemonstrationRecorder.cs

```diff
@@ -35,15 +36,16 @@ void Update()
         /// <summary>
         /// Creates demonstration store for use in recording.
         /// </summary>
-        void InitializeDemoStore()
+        public void InitializeDemoStore(IFileSystem fileSystem = null)
```

Comment:
Allow for the mock file system to be injected. Could keep this private and do it by reflection, but not sure if that's necessary.